### PR TITLE
[RFR] Removed Flash Messages and reordered imports

### PR DESCRIPTION
**Pull request by psav**
Removed Flash Messages and reordered imports

### CODE REVIEW

File: cfme/infrastructure/virtual_machines.py [hunk 1]
```diff
@@ -26,12 +22,18 @@
...
     VmOrInstanceNotFound)
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.utils import version
 from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
 from cfme.utils.conf import cfme_data
 from cfme.utils.log import logger
 from cfme.utils.pretty import Pretty
 from cfme.utils.wait import wait_for
-from cfme.utils import version
+from cfme.web_ui import (
```
Comment (mshriver) [inline cfme/infrastructure/virtual_machines.py]: This should be wt_manageiq Table import

Comment (psav) [reply cfme/infrastructure/virtual_machines.py]: It hsould indeed - merge issue

File: cfme/web_ui/mixins.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # -*- coding: utf-8 -*-
-from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import fill, Form, AngularSelect, Table, toolbar, form_buttons, flash
 from xml.sax.saxutils import quoteattr
+
+from cfme.fixtures import pytest_selenium as sel
 from cfme.utils.version import current_version
+from cfme.web_ui import fill, Form, AngularSelect, Table, toolbar, form_buttons
```
Comment (mshriver) [inline cfme/web_ui/mixins.py]: web_ui.Form was removed in #6150

Comment (mshriver) [reply cfme/web_ui/mixins.py]: mixin is used in one test module, cloud_infra_common.test_tag_objects.

Need to update that test module, and then this mixin can be removed

Comment (mshriver) [reply cfme/web_ui/mixins.py]: This is in #6154 actually

**Pull request merged**